### DISCUSSION TITLE
Add M1 guide to wiki

### DISCUSSION THREAD

**Discussion by SichangHe** [asmvik/yabai#1177] [Q&A] [answered]
# Add M1 guide to wiki
## wiki not complete on M1
Installing yabai on M1 machines require extra steps.
In the [installation wiki](https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)) page, it says
> (NOTE: The scripting addition does currently not support Apple Silicon (M1).)

which is confusing because the version from HEAD supports M1
The [Installing yabai (from HEAD)](https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(from-HEAD)) supposedly support M1 users, but the guide misses some parts, which confused me in #1158 
## addition needed
before
> Now install the scripting addition.

we need
```shell
sudo nvram boot-args=-arm64e_preview_abi
```
### suggestion
Just add a new wiki page for M1 users. Installation for M1 is far different.

**Comment by asmvik** [comment, marked as answer]
It is outlined here in the first step under installation: https://github.com/koekeishiya/yabai/wiki/Disabling-System-Integrity-Protection

The latest release is v3.3.10 and it does not support M1, as stated in the wiki. The dev branch does support M1/Monterey to some extent.
